api_spec.yaml

```diff
@@ -322,12 +323,18 @@ paths:
                       $ref: '#/components/schemas/createJWTPayload'
                   required:
                     - authorizationCode
```

Comment:
in the request body, where we give the refresh token and the client id, we should not pass in the access and refresh token lifetime. The new token's lifetimes should be the same as what was for the previous access / refresh token for that session.

Reply:
Yes , doesn't make sense to pass new tokens life time here. Will check and update.

Reply:
Discused this @porcellus . We are allowing modify of payload and lifetime details of access tokens and refresh tokens in refresh flow.

Reply:
So these fields are required.